docs/user-guide/extra_info.md

```diff
@@ -0,0 +1,28 @@
+# Add extra Application info
```

Comment:
Can you.add this page to the mkdocs.yml table of contents?

Reply:
Done!